README.md

```diff
@@ -1 +1,58 @@
-# morse_decode
\ No newline at end of file
+# morse_decode
+Using Ruby language to decode a Morse code
+
+## Built With
+- Ruby
+
+## Preview
+![screenshot](https://commons.wikimedia.org/wiki/File:International_Morse_Code.svg#/media/File:International_Morse_Code.svg)
+
+## Set-up Rubocop GitHub Action
+
+[Rubocop](https://www.rubocop.org/) is a Ruby static code analyzer (a.k.a. linter) and code formatter. It will enforce many of the guidelines outlined in the community [Ruby Style Guide](https://rubystyle.guide/).
+
+## Set-up linters in your local env
+
+### [RuboCop](https://docs.rubocop.org/en/stable/)
+
+1. Add this line to the `Gemfile`
+    ```
+    gem 'rubocop', '>= 1.0', '< 2.0'
+    ```
+    *not sure how to use Gemfile? Read [this](https://bundler.io/v1.15/guides/bundler_setup.html).*
+2. Run `bundle install`.
+3. Copy [.rubocop.yml](./.rubocop.yml) to the root directory of your project
+5. Run `rubocop`.
+6. Fix linter errors.
+
```

Comment:
- [ ] OPTIONAL
Kindly add instruction in the ReadMe on how to run your app. This will make your ReadMe appear more professional.